integration/integration_test.go

```diff
@@ -0,0 +1,64 @@
+package integration
+
+import (
+	"testing"
+
+	"sigs.k8s.io/krew/integration/krew"
+)
+
+const (
+	// validPlugin is a valid plugin with a small download size
+	validPlugin = "konfig"
+)
+
+func TestInstall(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	krewTest, cleanup := krew.NewKrewTest(t)
+	defer cleanup()
+
+	krewTest.WithIndex().Cmd("install", validPlugin).RunOrFail()
+}
+
+func TestUpdate(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	krewTest, cleanup := krew.NewKrewTest(t)
+	defer cleanup()
+
+	krewTest.WithIndex().Cmd("update").RunOrFail()
+
+	indexFiles, err := krewTest.TempDir().List("index")
+	if err != nil {
+		t.Error(err)
+	}
+
+	if len(indexFiles) == 0 {
+		t.Error("expected some index files but found none")
+	}
+}
+
+func TestUninstall(t *testing.T) {
+	if testing.Short() {
```

Comment:
Sounds like you should use a t.Helper() for this ;-) 
A skipShort(t) can simplify. 

We’ll go through many syntax refactorings with the test utils in this pkg as we write more tests. But this seemed obvious to me.